build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Should this use `/usr/bin/env bash`? I'm not familiar with all the `env` fanciness, but I keep seeing it pop up when looking at script examples.

Reply:
env is used to create a custom environment when running a command. In this case, it would just invoke bash, with no additional custom environment (since there are no environment variables we need to define there).

Reply:
What happens if `bash` is not in `/bin`? (Can that happen? Does `env` solve that?)

Reply:
Env does not solve that. If bash is not in bin then the system is highly nonstandard and there are no guarantees that env is in /usr/bin